model/annotations/task_annotations.go

```diff
@@ -210,3 +218,36 @@ func UpdateAnnotation(a *TaskAnnotation, userDisplayName string) error {
 	)
 	return errors.Wrapf(err, "problem adding task annotation for '%s'", a.TaskId)
 }
+
+func GetJiraTickets(issuelinks []IssueLink) ([]DisplayTicket, error) {
+	var displayTickets []DisplayTicket
+
+	settings := evergreen.GetEnvironment().Settings()
+	jiraHandler := thirdparty.NewJiraHandler(*settings.Jira.Export())
+
+	for _, issue := range issuelinks {
+		displayTicket := DisplayTicket{}
+
+		urlObject, err := url.Parse(issue.URL)
+		if err != nil {
+			return nil, errors.Wrap(err, "problem parsing issue string")
+		}
+
+		if urlObject.Host != "jira.mongodb.org" {
+			displayTicket.issueData = issue
+			displayTickets = append(displayTickets, displayTicket)
+		}
+
+		jiraIssue, err := jiraHandler.GetJIRATicket(issue.IssueKey)
+		if err != nil {
+			return nil, err
```

Comment:
Similarly here, if Jira is down we should probably know about it, but we can still display the annotation without the ticket information.

Reply:
added a catcher